packages/domainproperties/src/models.ts

```diff
@@ -1392,3 +1463,5 @@ export interface IAppDomainHeader {
     onAddField?: (fieldConfig: Partial<IDomainField>) => void
     onDomainChange?: (index: number, updatedDomain: DomainDesign) => void
 }
+
+export type AssayPanelStatus = 'INPROGRESS' | 'TODO' | 'COMPLETE' | 'NONE';
```

Comment:
See note from other part of code review. Since this isn't assay specific, I suggest renaming to PanelStatus or DomainPanelStatus.